app/server.go

```diff
@@ -246,7 +246,7 @@ func NewServer(options ...Option) (*Server, error) {
 	// in the future the cache provider will be built based on the loaded config
 	s.CacheProvider = cache.NewProvider()
 	if err := s.CacheProvider.Connect(); err != nil {
-		return nil, errors.Wrapf(err, "Unable to connect to cache provider")
+		mlog.Error("We have an error here!", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://docs.google.com/document/d/160-9Qfk-1a2_gOU-VHfZ5CbUdkY8BZxq-2VL7XYwqGA) before merging this change.